202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md

```diff
@@ -79,6 +79,9 @@ Clickjacking is a possibility to tweak the UI or craft a malicious webpage, embe
 
 ### Obsolete or outdated dependencies
 Keep your dependencies up-to-date.
+Make sure that:
+*   You have all the security updates from Spryker modules.
+*   You have all the security updates from other modules.
```

Comment:
Could you give an example of a non-Spryker module?

Reply:
f.e Symfony